docs/metadata.md

```diff
@@ -9,32 +9,51 @@ How To:
 * [Additional Options](#additional-options)
 * [Running Additional Collections](#running-additional-collections)
 
-# How it Works
+## How it Works
 
 The metadata collection is done through the use of [Stockpile](https://github.com/cloud-bulldozer/stockpile), [Backpack](https://github.com/cloud-bulldozer/backpack) and [Scribe](https://github.com/cloud-bulldozer/scribe).
 The data is then uploaded to a defined Elasticsearch instance.
 
+Metadata collection is enabled by default, it can be disabled setting collection to false in the metadata section:
+
+```yaml
+metadata:
+  collection: false
+```
```

Comment:
I'm good with enabling metadata collection by default however would like to get @jtaleric , @bengland2 , @chaitanyaenr and @mohit-sheth 's thoughts as well